File: Seccion 12 - Punteros en C++/Seccion_12_Python/03_ejercicio_03.py

def main():
    numeros: list = []
    
    for i in range(10):
        valor = int(input('Digite un número: '))
        numeros.append(valor)
        
    for val in numeros:
        if val % 2 == 0:
            print('El número {} es par.'.format(val))

if __name__ == '__main__':
    main()

File: Seccion 12 - Punteros en C++/Seccion_12_Dart/03_ejercicio_03.dart

import 'dart:io';

void main() {

  List<int> numeros = new List();

  for (int i = 0; i < 10; i++) {
    stdout.write('Digite un número: ');
    numeros.add(int.parse(stdin.readLineSync()));
  }

  for (int i = 0; i < numeros.length; i++) {
    if (numeros[i] % 2 == 0) {
      print('El número ${numeros[i]} es par.');
    }
  }
}